README.md

```diff
@@ -104,6 +104,7 @@ Name | Website | Region
 [Brainstorm Force](/company-profiles/brainstorm-force.md) | https://brainstormforce.com/ | Worldwide
 [Brave Investments](/company-profiles/brave-investments.md)  | https://www.crunchbase.com/organization/brave-investment | Brazil
 [Bright Funds](/company-profiles/bright-funds.md) | https://www.brightfunds.org | USA
+[Brikl](/company-profiles/brikl.md) | https://www.brightfunds.org | North America, Asia
```

Comment:
I believe the URL listed is also a copy monster 😆

Reply:
fix, Thanks!! it seem that PR before coffee is not a good idea.